utils/hct/hctdb_instrhelp.py

```diff
@@ -546,55 +537,41 @@ def print_opfunc_props(self):
         oload_to_mask = lambda oload: sum(
             [1 << dxil_all_user_oload_chars.find(c) for c in oload]
         )
-        ext_oload_fn = (
-            lambda i: "{"
-            + ",".join(["{0x%x}" % oload_to_mask(o) for o in i.extended_oload_types])
-            + "}"
-        )
-        vec_oload_fn = (
-            lambda i: "{"
-            + ",".join(["{0x%x}" % oload_to_mask(o) for o in i.vector_oload_types])
-            + "}"
+        oloads_fn = lambda oloads: (
+            "{" + ",".join(["{0x%x}" % m for m in oloads]) + "}"
         )
         for i in self.instrs:
             if last_category != i.category:
                 if last_category != None:
                     print("")
-                print(
-                    "  // {category:118}  void,     h,     f,     d,    i1,    i8,   i16,   i32,   i64,   udt,   obj,   vec,  function attribute, ext oload, vec oload".format(
```

Comment:
While I hope it was clear that my previous comment wasn't demanding this change, I won't miss the giant boolean matrix.